code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -781,3 +781,9 @@
 	id = /datum/reagent/consumable/red_queen
 	results = list(/datum/reagent/consumable/red_queen = 10)
 	required_reagents = list(/datum/reagent/consumable/tea = 6, /datum/reagent/mercury = 2, /datum/reagent/consumable/blackpepper = 1, /datum/reagent/growthserum = 1)
+
+/datum/chemical_reaction/mauna_loa
+	name = "Mauna Loa"
+	id = /datum/reagent/consumable/ethanol/mauna_loa
+	results = list(/datum/reagent/consumable/ethanol/patron = 25)
```

Comment:
```suggestion
	results = list(/datum/reagent/consumable/ethanol/mauna_loa = 25)
```

Reply:
That seems like a good idea; must've forgot to change some stuff when copy pasting. Thanks!